static/src/js/util/map/normalizeGranuleSpatial.js

```diff
@@ -65,6 +128,26 @@ const divideLine = (line) => {
   return dividedCoordinates
 }
 
+// Return the given polygon as counter-clockwise.
+// Exterior rings of polygons should be counter-clockwise and holes should be clockwise
+const makeCounterClockwise = (polygon) => {
```

Comment:
makeCounterClockwise and makeClockwise can probably be combined into one function

Reply:
How would you structure that function? The ways I have thought to do that wouldn't read as clearly as two separate functions

Reply:
`const setPolygonWinding = (polygon, clockwise) => {
  return booleanClockwise(polygon) !== clockwise ? polygon.reverse() : polygon
}`

or 

`const setPolygonWinding = (polygon, clockwise) => {
  const isClockwise = booleanClockwise(polygon)
  
  if (isClockwise !== clockwise) {
    return polygon.reverse()
  }
  
  return polygon
};`

if you think its more readable with two separate functions I'm not against that, the code does read very similar too each other so it suggests dedup when reading it as a reviewer.